Sources/Brave/Frontend/Browser/Tabs/RecentlyClosedTabs/RecentlyClosedTabsView.swift

```diff
@@ -30,13 +31,14 @@ struct RecentlyClosedTabsView: View {
     .foregroundColor(Color(.braveBlurpleTint))
     .actionSheet(isPresented: $showClearDataPrompt) {
       .init(title: Text(Strings.RecentlyClosed.recentlyClosedClearActionConfirmation),
-        buttons: [
-          .destructive(Text(Strings.RecentlyClosed.recentlyClosedClearActionTitle), action: {
-            RecentlyClosed.removeAll()
-            dismissView()
-          }),
-          .cancel()
-        ]
+            buttons: [
```

Comment:
Can you fix the indentation here? we use 2 space indentation.

Reply:
@soner-yuksel 
I believe this re-indentation is correct but looks funny in the diff. The reason there is a change is because Xcode indentation is trying to line up the function parameters for the `init` which are `title` and `buttons`. Before these where not lined up. If you want these to be indented automatically on new line, I would need drop `title` parameter to a new line. Is that preferred?

(1) Current:
<img width="1000" alt="Screenshot 2023-08-23 at 10 29 56 AM" src="https://github.com/brave/brave-ios/assets/43318682/a076e0ef-5714-41d9-abeb-1978149fd7a7">

Alternative:
<img width="1000" alt="Screenshot 2023-08-23 at 10 33 26 AM" src="https://github.com/brave/brave-ios/assets/43318682/13411d10-ba6a-4e50-8ecd-9f446f981c61">